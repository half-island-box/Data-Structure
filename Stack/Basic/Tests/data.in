input data for testing.
1+2+3+4#
88-1*5#
1024/4*8#
1024/(4*8)#
(20+2)*(6/2)#
2*(6+2*(3+6*(6+6)))#
(((6+6)*6+3)*2+6)*2#
8/(9-9)#
