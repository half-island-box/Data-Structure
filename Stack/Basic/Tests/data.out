output data for testing
10
83
2048
32
66
312
312
0 can't be a denominator.
